wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -812,9 +838,8 @@ private boolean compareAndSetStatus(String id, WorkspaceStatus from, WorkspaceSt
     }
 
     /** Removes state from in-memory storage in write lock. */
-    private void removeState(String workspaceId) {
+    private void removeState(String workspaceId) throws ServerException {
```

Comment:
Can you point me why it throws ServerException, looks like I'm missed something.